### PR TITLE
HAL_ChibiOS: added input capture filter

### PR DESCRIPTION
**Pull request by sh83**
Added input filtering for RCInput timer.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/SoftSigReaderInt.cpp [hunk 1]
```diff
@@ -21,6 +21,24 @@
 using namespace ChibiOS;
 extern const AP_HAL::HAL& hal;
 
+void stm32_timer_set_capture_filter(stm32_tim_t *tim, uint8_t channel, uint8_t filter_mode)
```
Comment (sh83) [inline libraries/AP_HAL_ChibiOS/SoftSigReaderInt.cpp]: May be it should go to Utils.cpp/Utils.h ?

**Comment by tridge** [comment]
the extern void is a bit out of place. I'd put the function in hwdef/common/stm32_util.c, and add a hwdef/common/stm32_util.h 
That will give us a reasonable place to put other similar helper functions

**Comment by sh83** [comment]
Ok, will fix it

**Comment by sh83** [comment]
Moved function to stm32_util.h/.c

**Comment by tridge** [comment]
looks good, thanks!

**Pull request merged**